app/controllers/clients_controller.rb

```diff
@@ -1,2 +1,25 @@
 class ClientsController < ApplicationController
+  def show
+    client = Client
+      .where(email: [client_params[:email]])
```

Comment:
Could you please give me an example how does this work? 🤔 I don't get it :(

Reply:
I wanted to search for a particular Client but if the ID was not given the action is supposed to search for Client by email (emails are unique and every client knows her or his email). I'm not sure this approach is correct. I will see later during app development ;D

Reply:
Anyway, thanks for your question because now I can see the code doesn't work how it was supposed to. I thought I checked if I put correct ID of one user and email of another one and I get only one record but I receive two differentt users :/